actions/fetchUsers.js

```diff
@@ -10,19 +10,18 @@ export default function fetchUsers(args) {
 		dispatch({
 			type: 'USERS_UPDATING',
 		});
-		api.get('/wp/v2/users', args).then(function(data, err) {
-			if (err) {
-				return dispatch({
-					type: 'USERS_UPDATE_ERRORED',
-					payload: {
-						error: err,
-					},
-				});
-			}
+		api.get('/wp/v2/users', args).then(function(data) {
```

Comment:
nacin spacin etc

Reply:
Gunna re-format the whole project in a followup